windows-driver-docs-pr/develop/validating-windows-drivers.md

```diff
@@ -19,7 +19,65 @@ Use InfVerif with `/w` and `/v` to verify that a Windows Driver:
 * Meets the **declarative (D)** principle of [DCH Design Principles](dch-principles-best-practices.md)
 * Complies with the [driver package isolation](driver-isolation.md) requirement of [Getting Started with Windows Drivers](getting-started-with-windows-drivers.md)
 
-For more details, see [Running InfVerif from the command line](../devtest/running-infverif-from-the-command-line.md) for 
+For more details, see [Running InfVerif from the command line](../devtest/running-infverif-from-the-command-line.md).
+
+## Driver Verifier Driver Isolation Checks
+
+[Driver Verifier](https://docs.microsoft.com/windows-hardware/drivers/devtest/driver-verifier) (DV) is a tool that ships with Windows that is used to monitor drivers for improper function calls or actions that may corrupt the system.  Starting in Windows 10 Preview Builds 19568 and above, Driver Verifier has new functionality to support developers of Windows Drivers by monitoring for violations of Driver Isolation requirements.  Driver Verifier will monitor for improper registry reads and writes that are not allowed for isolated driver packages. Use this tool early on in driver development to understandand and fix where your component is violating driver isolation requirements.
+
+### Syntax
+
+```
+verifier /rc 33 36 /driver myDriver.sys
+```
+
+This will enable the driver isolation checks on your targeted driver (myDriver.sys).  You can also select multiple drivers by separating the list with a space:
+
+```
+verifier /rc 33 36 /driver myDriver1.sys myDriver2.sys
+```
+
+A reboot is required for the verification settings to be enabled.  You can do so by specifying via the command line:
+
+```
+shutdown /r /t 0
+```
+
+### Expected Behavior - Telemetry Mode
+
+During initial stages of driver bring-up, the **recommended behavior** for these checks is **telemetry mode**.This is the default behavior and will provide a way for developers to view all violations without a bugcheck disrupting progress. 
+
+It is recommended that DV driver isolation checks are enabled using the syntax specified in the section above with a kernel debugger attached.  **After a reboot has enabled the DV settings, you will be able to see violations in the kernel debugger output**.  
+
+Below are a couple example scenarios of a driver violating driver isolation requirements and what typical output would look like:
+
+*Scenario*: ZwCreateKey using full absolute path:
+
+```“DRIVER_ISOLATION_VIOLATION: <driver name>: Registry operations should not use absolute paths. Detected creation of unisolated registry key '\Registry\Machine\SYSTEM’”
```

Comment:
I don't see this in the rich diff view.  Is there something wrong with how this is specified or is the rich diff displaying things incorrectly?

Reply:
odd, can you check now? still not seeing it?

Reply:
Pro tip - if you open a PR in the private version of this repo (add -pr to the end of the repo name), you'll get a review page on docs.ms.com.  (Plus as an FTE you can create branches in the private)